src/atoms/DymoHeading.js

```diff
@@ -15,16 +16,16 @@ const PaddedText = styled.span`
 	 	margin: 0;
 		padding-top: 0;
 		box-shadow:
-			calc(1/2*${props => props.theme.variables.horizontalBase})
+			calc(1/2 * ${getVariable('horizontalBase')})
 			0
-		  ${props           => props.theme.colors[props.theme.colors.skinColors[props.skin.backgroundColor]]}
+		  ${props => getColor(props.skin.backgroundColor)}
 		;
-		background: ${props => props.theme.colors[props.theme.colors.skinColors[props.skin.backgroundColor]]};
-		color:      ${props => props.theme.colors[props.theme.colors.skinColors[props.skin.textColor]]};
+		background: ${props => getColor(props.skin.backgroundColor)};
+		color:      ${props => getColor(props.skin.textColor)};
```

Comment:
why do we do `${props => getColor(props.skin.textColor)};` is in not possible to do 
`${getColor(props.skin.textColor)}`;

if all the variables is uniq it is possible to also create const foreach so we get `${getColor(TEXT_COLOR)}`; or similar, but here we should include the designers!

Reply:
`getColor` expects a string that is defined either on the `skinColors` object or directly in the color palette. If we hard code the color, `${getColor('red')}` is fine. In this case, though, the color name is dynamically fetched from a prop. To access the props, we still need the `${props => props.myProp}` syntax.

Reply:
However, I'd say that still is a significant improvement from
```
${props => (
  props.theme.colors[props.theme.colors.skinColors[props.skin.backgroundColor]]
  || props.theme.colors[props.skin.backgroundColor]}
);
```